### PR TITLE
T1113606: Fix table attributors keyNames collision.

### PR DESCRIPTION
**Pull request by williamvinogradov**
Source ticket: [T1113606](https://supportcenter.devexpress.com/internal/ticket/details/T1113606)

Implemented the new mechanism of separating table 'attributors' from the others.

### CODE REVIEW

File: test/unit/blots/inline.js [hunk 1]
```diff
@@ -26,4 +27,68 @@ describe('Inline', function () {
       '<p><em>0</em><strong><em>12</em></strong><em>3</em></p>',
     );
   });
+
+  fdescribe('Strikethrough', function () {
```
Comment (ksercs) [inline test/unit/blots/inline.js:31]:
```suggestion
  describe('Strikethrough', function () {
```

File: test/unit/blots/inline.js [hunk 2]
```diff
@@ -26,4 +27,68 @@ describe('Inline', function () {
...
+    it('renders attributes when strikethrough tag the first', function () {
+      const testHtml = `
+      <p>
+        <s style="background-color: #00ff00">BBB</s>
+        <span style="background-color: #00ff00">CCC</span>
+      </p>`.replace(/(\n|>\s+|\s+<)/g, (match) => match.trim());
```
Comment (ksercs) [inline test/unit/blots/inline.js:37]: Let's move the RegExp to a constant to make code clearer.

File: test/unit/blots/inline.js [hunk 3]
```diff
@@ -26,4 +27,68 @@ describe('Inline', function () {
       '<p><em>0</em><strong><em>12</em></strong><em>3</em></p>',
     );
   });
+
+  fdescribe('Strikethrough', function () {
+    it('renders attributes when strikethrough tag the first', function () {
```
Comment (ksercs) [inline test/unit/blots/inline.js:32]: And in another places
```suggestion
    it('renders attributes when strikethrough tag is first', function () {
```

Comment (williamvinogradov) [reply test/unit/blots/inline.js]: Fixed

File: test/unit/blots/inline.js (hunk 2 above)
Comment (williamvinogradov) [reply test/unit/blots/inline.js:39]: Agree with you

File: test/unit/blots/inline.js (hunk 1 above)
Comment (williamvinogradov) [reply test/unit/blots/inline.js]: Fixed

**Comment by williamvinogradov** [comment]
[HTMLEditor - Style doesn't apply to the <s> html element when markup is defined via the value option](https://trello.com/c/DQgG4iy6/1920-htmleditor-style-doesnt-apply-to-the-s-html-element-when-markup-is-defined-via-the-value-option)

**Review by ksercs — APPROVED** [review]

**Pull request merged**